packages/core/apps/core-apps-server-internal/src/bundle_routes/register_bundle_routes.test.ts

```diff
@@ -56,31 +69,50 @@ describe('registerBundleRoutes', () => {
       uiPlugins: createUiPlugins(),
     });
 
-    expect(registerRouteForBundleMock).toHaveBeenCalledTimes(3);
-
-    expect(registerRouteForBundleMock).toHaveBeenCalledWith(router, {
-      fileHashCache: expect.any(FileHashCache),
-      isDist: true,
-      bundlesPath: 'uiSharedDepsSrcDistDir',
-      publicPath: '/server-base-path/42/bundles/kbn-ui-shared-deps-src/',
-      routePath: '/42/bundles/kbn-ui-shared-deps-src/',
-    });
-
-    expect(registerRouteForBundleMock).toHaveBeenCalledWith(router, {
-      fileHashCache: expect.any(FileHashCache),
-      isDist: true,
-      bundlesPath: 'uiSharedDepsNpmDistDir',
-      publicPath: '/server-base-path/42/bundles/kbn-ui-shared-deps-npm/',
-      routePath: '/42/bundles/kbn-ui-shared-deps-npm/',
-    });
-
-    expect(registerRouteForBundleMock).toHaveBeenCalledWith(router, {
-      fileHashCache: expect.any(FileHashCache),
-      isDist: true,
-      bundlesPath: expect.stringMatching(/src\/core\/target\/public/),
-      publicPath: '/server-base-path/42/bundles/core/',
-      routePath: '/42/bundles/core/',
-    });
+    expect(registerRouteForBundleMock.mock.calls).toMatchInlineSnapshot(`
```

Comment:
Why where our proper `toHaveBeenCalledWith` assertions replaced with ugly and long `toMatchInlineSnapshot`?

Reply:
Okay, okay, I disagree that these are improper, but my laziness and love for automatically updating snapshots is not something I'll force on the core team 😝